pkg/pipelines/argocd/argocd.go

```diff
@@ -52,32 +51,38 @@ func Build(argoNS, repoURL string, m *config.Manifest) (res.Resources, error) {
 
 type argocdBuilder struct {
 	repoURL string
-	argoEnv *config.Environment
+	argoEnv *config.Argo
```

Comment:
This should probably be renamed as it's no longer an `argoEnv` ?

Reply:
Now changed to ArgoCDConfig and PipelinesConfig.